mne/annotations.py

```diff
@@ -0,0 +1,79 @@
+# Authors: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from datetime import datetime
+import time
+import json
+
+
+class Annotations(object):
+    """Annotation object for annotating segments of raw data.
+
+    Parameters
+    ----------
+    onset: array of float, shape (n_annotations,)
+        Annotation time onsets from the beginning of the recording.
```

Comment:
I originally did it for optimising ``raw.plot``. First I tried only drawing the segments once at the beginning, but noticed some weird artifacts with some settings. I couldn't find out the reason for them, so I suspect it's a bug in mpl. So I changed it so that the segments are plotted with every shift of the viewport and thus, you don't have to go through the whole list every time.
I'm not sure this is the best solution though.